jcl/src/java.base/share/classes/java/lang/Throwable.java

```diff
@@ -440,7 +440,7 @@ private void readObject(ObjectInputStream s)
 	if (suppressedExceptions != null) {
 		List<Throwable> newList = Collections.EMPTY_LIST;;
 		try {
-/*[IF Sidecar18-SE-OpenJ9&!Sidecar19-SE-OpenJ9]*/
+/*[IF Sidecar18-SE&!Sidecar19-SE-OpenJ9]*/
```

Comment:
Is an explicit `Sidecar18-SE` needed? It is the default `JPP` flag supported.

Reply:
The condition should just be `! Sidecar19-SE`. Better yet, invert the `IF`.

Reply:
I don't understand this comment.

Reply:
Take a look at more of the code, there is an [ELSE] statement

Reply:
Ok, I understand. It might work to just use `!Sidecar19-SE-OpenJ9`, I didn't try it. I don't think it matters too much as we are planning to replace all these statements in the future, like `[IF JAVA_SPEC_VERSION == 8]`